src/logger.ts

```diff
@@ -43,8 +45,11 @@ export function sendCrashResponse({
   // right before sending the response, to make sure that no concurrent
   // execution sends the response between the check and 'send' call below.
   if (res && !res.headersSent) {
-    res.set(FUNCTION_STATUS_HEADER_FIELD, 'crash');
-    res.send((err.message || err) + '');
+    res.set(FUNCTION_STATUS_HEADER_FIELD, statusHeader);
+    res.status(500);
+    if (process.env.NODE_ENV !== 'production') {
+      res.send((err.message || err) + '');
```

Comment:
Can we log the error message for production so the end user can still find the info?

May also be worth including a descriptive response body message that's generic like "error encountered during function execution, see logs for more details."

Reply:
stack trace is still logged (as it was before) up on line 40.